src/Perpetuum/Services/Mail/MailHandler.cs

```diff
@@ -185,5 +185,39 @@ public static int NewMailCount(Character character)
         {
             return Db.Query().CommandText("newMailCount").SetParameter("@characterID", character.Id).ExecuteScalar<int>();
         }
+
+        public static ErrorCodes SendWelcomeMailBeginTutorial(Character newPlayer)
+        {
+            return SendWelcomeMail(newPlayer, 1);
+        }
+
+        public static ErrorCodes SendWelcomeMailExitTutorial(Character newPlayer)
+        {
+            return SendWelcomeMail(newPlayer, 2);
+        }
+
+        private static ErrorCodes SendWelcomeMail(Character newPlayer, int mailId)
+        {
+            // TODO: Change the sender character name for something more/less appropriate
+            Character sender = Character.GetByNick("overlord");
```

Comment:
The current character name is a placeholder.

Reply:
Be sure to update when a name is agreed to!
I'll put the question to the team